obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -35,18 +35,19 @@ RUN curl -s -L -O https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x8
     conda config --set show_channel_urls True && \
     conda update --all --yes && conda clean -tipsy
 
-# Add conda and friends to the path.
-ENV PATH /opt/conda/bin:$PATH
-
 # Install Obvious-CI.
-RUN conda install --yes -c pelson/channel/development obvious-ci && \
+RUN export PATH="/opt/conda/bin:${PATH}" && \
```

Comment:
So, not sure if you are aware of this, if you are sorry for explaining the obvious.

The Docker `RUN` command is literally like running `/bin/sh -c "..."`. Not quite because it will fix the quotes for you, but basically.

This means even though we exported this here. It will be gone once we leave this `RUN` command.